libminifi/src/capi/Plan.cpp

```diff
@@ -115,7 +115,7 @@ bool linkToPrevious) {
 
 bool ExecutionPlan::setProperty(const std::shared_ptr<core::Processor> proc, const std::string &prop, const std::string &value) {
   uint32_t i = 0;
-  logger_->log_info("Attempting to set property %s %s for %s", prop, value, proc->getName());
+  logger_->log_debug("Attempting to set property %s %s for %s", prop, value, proc->getName());
```

Comment:
oh man this is definitely me! I copied and pasted a lot of these. Thanks for catching this!